docs/source/documentation.rst

```diff
@@ -32,55 +32,6 @@ Installation
-
-First, verify if you already have an AxonDeepSeg-compatible version of Miniconda or Anaconda properly installed and is in your systems path. 
-
-In a new terminal window (macOS or Linux) or Anaconda Prompt (Windows – if it is installed), run the following command:::
-
-    conda search python
-
-If a list of available Python versions are displayed and versions >=3.8.0 are available, you may skip to the next section (git).
-
-Linux
-~~~~~
-
-To install Miniconda, run the following commands in your terminal:::
-
-    cd ~
-    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh
-    bash ~/miniconda.sh -b -p $HOME/miniconda
-    echo ". ~/miniconda/etc/profile.d/conda.sh" >> ~/.bashrc
-    source ~/.bashrc
-
-.. NOTE :: If ``conda`` isn't available on new terminal sessions after running these steps, it's possible that your system is configured to use a different startup script. Try adding the line ``source ~/.bashrc`` to your ``~/.bash_profile`` file. `See here <http://www.joshstaiger.org/archives/2005/07/bash_profile_vs.html>`_ for more details.
-
-macOS
-~~~~~
-
-To install Miniconda, run the following commands in your terminal:::
-
-    cd ~
-    curl https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -o ~/miniconda.sh
-    bash ~/miniconda.sh -b -p $HOME/miniconda
-    echo ". ~/miniconda/etc/profile.d/conda.sh" >> ~/.bash_profile
-    source ~/.bash_profile
-
-Windows
-~~~~~~~
-
-.. NOTE :: The AxonDeepSeg installation instruction using the Miniconda have only been tested for Windows 10. Older versions of Windows may not be compatible with the tools required to run AxonDeepSeg.
-
-To install Miniconda, go to the `Miniconda installation website <https://conda.io/miniconda.html>`_ and click on the Python 3.x version installer compatible with your Windows system (64 bit recommended). After the download is complete, execute the downloaded file, and follow the instructions. If you are unsure about any of the installation options, we recommend you use the default settings.
-
-git
----
-``git`` is a software version control system. Because AxonDeepSeg is hosted on GitHub, a service that hosts ``git`` repositories, having ``git`` installed on your system allows you to download the most up-to-date development version of AxonDeepSeg from a terminal, and also allows you to contribute to the project if you wish to do so.
-
-To install ``git``, please follow instructions for your operating system on the `git website <https://git-scm.com/downloads>`_
-
 Install AxonDeepSeg
```

Comment:
The sentence below this header mentions "Anaconda prompt" for Windows, but we can change this to say "Command Prompt" now, since users no longer have to install Anaconda. :)

Reply:
Good catch!